assets/css/less/front-end/elements/buttons.less

```diff
@@ -19,18 +19,78 @@ body .wpbdp-excerpt {
 		display: inline-flex;
 		align-items: center;
 		justify-content: center;
+		border: 1px solid;
+		border-radius: 4px;
```

Comment:
Let's remove this line. I think we should let the theme handle the border radius so the buttons will match as closely as possible.

Without this line, here's what I see in Divi. The secondary button has the .et_pb_button class, but the main button doesn't have a class from the theme. Perhaps we need to apply the button class to the input submit too?
<img width="231" alt="Screenshot 2023-08-10 at 6 06 42 PM" src="https://github.com/Strategy11/business-directory-plugin/assets/1116876/835f6490-aa86-49fd-97e1-e56b1b10890a">

Reply:
Done